Code/Tools/ProjectManager/Resources/ProjectManager.qss

```diff
@@ -710,7 +710,7 @@ QTabBar::tab:focus {
 }
 
 #projectButton QLabel#otherEngineLabel {
-    color: #dd0;
+    color: #ddd;
```

Comment:
So the color here is now #dddddd.
So a very light grey?

Reply:
yes, very close to white, but not pure white